### DISCUSSION TITLE
Image upload fails with 504

### DISCUSSION THREAD

**Discussion by leanzero-srl** [keystonejs/keystone#8701] [Questions]
We have a nextjs frontend application that connects to the keystonejs backend. Uploading files through the nextjs frontend application seems to fail while doing it from the admin application works just fine.

Our application is hosted on AWS and uses a loadbalancer to deliver https traffic. Has anyone encountered anything like this?

**Comment by dcousens** [comment]
@mperdum do you want to share how you are uploading?
